### PR TITLE
Fixing R examples in documentation :: Tutorials page

### PR DESCRIPTION
**Pull request by mparkhe**
## What changes are proposed in this pull request?
 
- Fixed `r_wine/train.R` to include `crate` dependency and package `model` 
- R code examples in documentation (Tutorials page)
 
## How is this patch tested?
 
(Details)
 
### Is this a user-facing change? 

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.
 
(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)
 
### What component(s) does this PR affect?
 
- [ ] UI
- [ ] CLI 
- [ ] API 
- [ ] REST-API 
- [x] Examples 
- [x] Docs
- [ ] Tracking
- [ ] Projects 
- [ ] Artifacts 
- [ ] Models 
- [ ] Scoring 
- [ ] Serving
- [x] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:
 
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by stbof** [comment]
LGTM.

File: examples/r_wine/train.R [hunk 1]
```diff
@@ -26,7 +27,7 @@ lambda <- mlflow_param("lambda", 0.5, "numeric")
 
 with(mlflow_start_run(), {
     model <- glmnet(train_x, train_y, alpha = alpha, lambda = lambda, family= "gaussian", standardize = FALSE)
-    predictor <- crate(~ glmnet::predict.glmnet(model, as.matrix(.x)), model)
+    predictor <- crate(~ glmnet::predict.glmnet(!!model, as.matrix(.x)), !!model)
```
Comment (tomasatdatabricks) [inline examples/r_wine/train.R]: what are the '!!' for?

**Review by tomasatdatabricks — COMMENTED** [review]
looks good, I had one question other than that lgtm

File: examples/r_wine/train.R (hunk 1 above)
Comment (mparkhe) [reply examples/r_wine/train.R]: `!!` is needed by `crate` to capture local state, in this case `model`
https://www.rdocumentation.org/packages/carrier/versions/0.1.0/topics/crate

**Pull request merged**